crates/nu-protocol/src/config/table.rs

```diff
@@ -60,6 +62,7 @@ impl ReconstructVal for TableMode {
                 TableMode::Compact => "compact",
                 TableMode::WithLove => "with_love",
                 TableMode::CompactDouble => "compact_double",
+                TableMode::Default => "rounded",
```

Comment:
If it reports `"rounded"` anyways, you could just drop `TableMode::Default` entirely and only add the line `"default" => Ok(TableMode::default())` to the `impl FromStr`. Then there is no chance for that to get out of step in `nu-table`'s `load_theme()`

Reply:
ok, did that too. i sincerely hope you're out of ideas of other things to change here. 🤣

Reply:
I'm done 😄